### ISSUE TITLE
Wrong package listed in setup.py

### ISSUE THREAD

**Issue by Murnto** [alexk307/redis_cache#12]
https://github.com/alexk307/cache_deco/blob/75956ee5cf97959dc702120eca6da981a2eac999/setup.py#L5

**Comment by alexk307** [comment]
Yeah, I renamed this project from `redis_cache` to `cache_deco`, this has to be changed.
